task/02-numbers-tasks.js

```diff
@@ -89,7 +89,8 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+    let result = -b / a;
+    return result === 0 ? 0 : result;
```

Comment:
We can just `return -b / a` because 
-0 === 0  (true)

Reply:
When I returning -b / a it says that 0 was expected but actually got -0 and test fails

Reply:
> We can just `return -b / a` because
> -0 === 0 (true)

Yes, I have checked this when I was doing the task, but when I return -b / a it says that 0 was expected but actually got -0 and test fails.

Reply:
Perhaps, it because of node version. Try to update node
![Selection_036](https://user-images.githubusercontent.com/26328278/90772557-678b3100-e2fd-11ea-9e06-2b56f1d3978f.png)

Reply:
But we have successful tracis build for such solution here:
https://github.com/AisBrestEDU/js-assignments/pull/928

https://travis-ci.org/github/AleshaKolesnikov/js-assignments/builds/719550067
Please, try to find differences.
May be node versions, travis.yml